include/nbl/builtin/hlsl/subgroup/scratch.hlsl

```diff
@@ -0,0 +1,114 @@
+// Copyright (C) 2023 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+#ifndef _NBL_BUILTIN_HLSL_SUBGROUP_SCRATCH_INCLUDED_
+#define _NBL_BUILTIN_HLSL_SUBGROUP_SCRATCH_INCLUDED_
+
+#include "nbl/builtin/hlsl/workgroup/basic.hlsl"
+#include "nbl/builtin/hlsl/subgroup/basic_portability.hlsl"
+
+/**
+ * The clearIndex for each invocation is the first 8 indices for each set of subgroup::Size().
+ * Example for subgroup::Size() == 32:
+ *	invocationId -> clearIndex
+ *	1 -> 1
+ *	7 -> 7
+ *	8 -> 32
+ *	15 -> 39
+ *	16 -> 64
+ *	24 -> 96
+ */
```

Comment:
whole file can be nuked thanks to ubiquitous support for `shuffle` subgroup feature

Reply:
don't see it being deleted in the pushed commits

Reply:
the file is still around and shouldn't be